src/wp-includes/compat.php

```diff
@@ -539,6 +539,46 @@ function array_all( array $array, callable $callback ): bool { // phpcs:ignore U
 	}
 }
 
+if ( ! function_exists( 'array_first' ) ) {
+	/**
+	 * Polyfill for `array_first()` function added in PHP 8.5.
+	 *
+	 * Returns the first element of an array.
+	 *
+	 * @since 6.8.3
```

Comment:
These will be added in the next major version but won't be backported to the current 6.8 branch.

```suggestion
	 * @since 6.9.0
```